src/hourly-operating-workspace/hourly-operating.service.ts

```diff
@@ -67,38 +71,53 @@ export class HourlyOperatingWorkspaceService {
       params,
     );
 
+    let resultPromises = [];
+
     if (hourlyOperating) {
-      const hourlyOperatingIds = hourlyOperating.map(i => i.id);
-      if (hourlyOperatingIds?.length > 0) {
-        const values = await Promise.all([
-          this.monitorHourlyValueService.export(hourlyOperatingIds),
-          this.derivedHourlyValueService.export(hourlyOperatingIds),
-          this.matsMonitorHourlyValueService.export(hourlyOperatingIds),
-          this.matsDerivedHourlyValueService.export(hourlyOperatingIds),
-          this.hourlyGasFlowMeterService.export(hourlyOperatingIds),
-          this.hourlyFuelFlowService.export(hourlyOperatingIds),
-        ]);
+      const hourlyOperatingDataChunks = await splitArrayInChunks(
```

Comment:
splitArrayInChunks doesn't seem to be an async function. We don't need that await there

Reply:
I put it because there was more than 69k issues. just the chunking process to be completed before running the db query. Let me test it by removing it.